Resources/Prototypes/DeltaV/Entities/Structures/Doors/airlock_groups.yml

```
@@ -0,0 +1,38 @@
+# Door resprites
+- type: AirlockGroup
+  id: StandardDeltaV
+  iconPriority: 200 # higher priority for spray painter
+  stylePaths:
+    atmospherics: DeltaV/Structures/Doors/Airlocks/Standard/atmospherics.rsi
+    basic:       DeltaV/Structures/Doors/Airlocks/Standard/basic.rsi
+    cargo:       DeltaV/Structures/Doors/Airlocks/Standard/cargo.rsi
+    chemistry:   DeltaV/Structures/Doors/Airlocks/Standard/chemistry.rsi 
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)